pkgs/by-name/cw/cwe-client-cli/package.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, rustPlatform, fetchFromGitHub, webkitgtk, gtk3, cairo, gdk-pixbuf
+, pkg-config, glib, dbus, openssl_3, librsvg, libsoup }:
+let
+  libraries = [
+    webkitgtk
+    gtk3
+    cairo
+    gdk-pixbuf
+    pkg-config
+    glib
+    dbus
+    openssl_3
+    librsvg
+    libsoup
+  ];
+in rustPlatform.buildRustPackage {
+  pname = "cwe-client-cli";
+  version = "0.2.0";
+  buildInputs = libraries;
+  nativeBuildInputs = libraries;
```

Comment:
There's no need to duplicate all `buildInputs` in `nativeBuildInputs`. It's best to drop the `libraries` list and have two separate lists: one for `buildInputs` for most of your dependencies, and a smaller one for `nativeBuildInputs`, which only has the dependencies that will be run in the build itself, such as `pkg-config`.